modules/nf-core/cadd/environment.yml

```diff
@@ -5,5 +5,5 @@ channels:
   - defaults
 dependencies:
   - bioconda::cadd-scripts=1.6.post1
-  - anaconda::conda=4.14.0
+  - conda-forge::conda=4.14.0
   - conda-forge::mamba=1.4.0
```

Comment:
Conda and mamba in one recipe? Why?

Reply:
@ramprasadn can you answer this since you wrote the module?